src/app/(app)/(home)/search-filters/categories.tsx

```
@@ -0,0 +1,29 @@
+import { Category } from "../../../../../payload-types"
+import { CategoryDropdown } from "./category-dropdown"
+
+interface Props {
+    data: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)